composer.json

```
@@ -82,6 +82,7 @@
         "symfony/http-foundation": "self.version",
         "symfony/http-kernel": "self.version",
         "symfony/intl": "self.version",
+        "symfony/intl-emoji": "self.version",
```

alpha order and this:
```suggestion
        "symfony/emoji": "self.version",
```